presentation/main_view.py

```diff
@@ -6,7 +6,7 @@
 import dash_html_components as html
 from dash.dependencies import Input, Output
 
-from domain.config import Config
+from domain.config.config import Config
```

Comment:
(nit) If you put `from config import Config` in `domain.config.__init__.py`, you won't need the funny looking `domain.config.config` path in the import statement.
It is IMHO a good practice to put "public" importable objects in `__init__.py` in each module.